📝(git) add guideline for the commit message of a new release (#7)
openfun/handbook: git.md

3. Update the project's `Changelog` following the [keepachangelog](https://keepachangelog.com/en/0.3.0/) recommandations,

> 4. Commit your changes with a structured message, add a title including the name of the release like: `Bump release to 4.18.1`, add to the body all the changes from the changelog concerned by this release, like :

Reviewer: ```suggestion
4. Commit your changes with a structured message:
- add a title including the name of the release like and respecting the above described format using the release gitmoji,
- paste in the body all the changes from the changelog concerned by this release, removing only the markdown tags and making sure that lines are shorter than 74 characters to satisfy gitlint.
```

Author: this is changed :-)

```

> Added :
> 
>  - Implement base CLI commands (list, extract, fetch & push) for supported
>  backends
>  - Support for ElasticSearch database backen
>  - Support for LDP storage backend
>  - Support for FS storage backend
>  - Parse (gzipped) tracking logs in GELF format

Reviewer: ```suggestion
🔖(minor) Bump release to 4.18.1

Added :

 - Implement base CLI commands (list, extract, fetch & push) for supported
 backends
 - Support for ElasticSearch database backen
 
 Changed:
 - Replace LDP storage backend by FS storage backend
```

Author: Your exemple has been added.

3. Update the project's `Changelog` following the [keepachangelog](https://keepachangelog.com/en/0.3.0/) recommandations,

4. Commit your changes with a structured message:

> - add a title including the name of the release like and respecting the above described format using the release gitmoji,

Reviewer 2: ```suggestion
- add a title including the version of the release and respecting the above described format using the 🔖 release emoji,
```

Author: Changes have been done.

> 

Reviewer 2: ```suggestion
```

> - paste in the body all the changes from the changelog concerned by this release, removing only the markdown tags and making sure that lines are shorter than 74 characters to satisfy gitlint.

Reviewer 2: ```suggestion
- paste in the body all changes from the changelog concerned by this release, removing only the markdown tags and making sure that lines are shorter than 74 characters.
```

Author: Changes have been done.

```
🔖(minor) Bump release to 4.18.0

> Added :

Reviewer 2: ```suggestion
Added :

```

Author: Changes have been done.

 - Implement base CLI commands (list, extract, fetch & push) for supported
 backends

>  - Support for ElasticSearch database backen

Reviewer 2: ```suggestion
 - Support for ElasticSearch database backend
```

>  Changed:

Reviewer 2: ```suggestion
 Changed:
 
```

3. Update the project's `Changelog` following the [keepachangelog](https://keepachangelog.com/en/0.3.0/) recommandations,

> 4. Commit your changes with a structured message, add a title including the name of the release like: `Bump release to 4.18.1`, add to the body all the changes from the changelog concerned by this release,

Reviewer: maybe paste here an example of release commit?

Author: I've added an exemple.
